sematic/ui/package.json

```diff
@@ -10,7 +10,8 @@
     "lint": "npm run lint --workspaces --if-present",
     "cypress:open": "cypress open --project packages/tests",
     "cypress:e2e": "cypress run --headless --e2e --project packages/tests",
-    "cypress:component": "cypress run --headless --component --project packages/tests"
+    "cypress:component": "cypress run --headless --component --project packages/tests",
+    "storybook": "npm run storybook --workspace @sematic/storybook"
```

Comment:
My understanding is that this application helps you develop, somehow. Please bear with me.
Why does it need all these files copied over manually, instead of added as dependencies? Do they hold the state of your experiments? Because only final shippable products should be committed.

Reply:
No worries. Can you help explain what are the files being copied over?

Reply:
Everything in the `sematic/ui/packages/storybook` directory. For example `Typography.stories.tsx`.

Reply:
`Typography.stories.tsx` is not a file copied over, it is a new file, which is what builds the storybook page as shown in the recording. 

Everything under `sematic/ui/packages/storybook` is a new package, and it does not merge into the `main` package, which is our dashboard app.